### ISSUE TITLE
Open Project in Browser Tab

### ISSUE THREAD

**Issue by bhavik2936** [bhavik2936/speed-reader#2]
## Description
As a developer, I want to have this process automated.

On `yarn start`, I want project URL to be opened in browser window under specific conditions.
- Open URL in Browser (i.e. default/mentioned)
- In specified mode of browser (i.e. incognito/guest)
- Above configurations can be customized for each developer (preferably via .env file)

## Implementation Details
Configure `dotenv` dependency.
Configure browser options in `webpack.config.js` via variables set in `.env` file.

**Comment by heyyayesh** [comment]
Hey! I want to give this a go, but I don't see a contribution guide. So I'm going to follow the standard contribution workflow. If there are any specific conventions, please let me know.

**Comment by bhavik2936** [comment]
Sure thing! Just follow PR template, no other restrictions.

**Comment by bhavik2936** [comment]
Hi @heyyayesh! Let me know if you're working on it or stuck, so we can resolve as well. Just to have clarity that issue is either in In-Progress or Ready-to-Pick.

**Comment by heyyayesh** [comment]
Yeah, my bad. I tried to solve but never really got around to do it. I should have mentioned that way earlier. So to clarify, I am no longer working on this issue. Apologies for any inconveniences caused.
